### PR TITLE
colin.hillis/doc 5360/about the docs new

### PR DESCRIPTION
**Pull request by colin-hillis**
New "About the Docs' page added, which includes:

1. How Enterprise features are identified.
2. EOL strategy.
3. How to report a docs error.
4. Reorganised section pages into more logical order.

Enterprise tooltip link to item 1 to follow.

### CODE REVIEW

File: content/en/platform/corda/5.0/introduction/about-the-docs.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Use the following table to track the end of life schedule for each version of Corda 5. Each version of Corda has R3 support available for a fixed period. 
+After this period has ended, these versions are no longer supported by R3 and associated documentation is archived. You should always aim to upgrade to the latest version of Corda whenever possible.
+
+Definitions:
+
+* **End of maintenance**: This release will no longer receive functional patches after the date shown.
+* **End of security**: This release will no longer be eligible for security patches after the date shown.
+* **End of support**: Support (including documentation) provided by R3 is no longer available after this date.
+
+| **Corda 5.X** | **Date of release** | **End of Maintenance** | **End of security**   | **End of support**    |
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/introduction/about-the-docs.md:42]:
```suggestion
| **Version** | **Date of Release** | **End of Maintenance** | **End of Security**   | **End of Support**    |
```

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.0/introduction/enterprise-icon.png
Comment (yarivus) [inline content/en/platform/corda/5.0/introduction/enterprise-icon.png:1]: I think you should rename this file ... it's not the enterprise-icon itself ...
maybe:
- enterprise-icon-screenshot
- enterprise-icon-explanation-screenshot

File: themes/doks/layouts/shortcodes/enterprise-icon.html [hunk 2]
```diff
@@ -9,6 +9,6 @@
     background-color: transparent !important;
   }
 </style>
-<a {{ with .Get "noMargin" }} style="margin-left: 0 !important;" {{ end }} title="This feature is only available in the Enterprise edition of Corda." href="#detailedlinkgoeshere" class="enterprise-icon">
+<a {{ with .Get "noMargin" }} style="margin-left: 0 !important;" {{ end }} title="This feature is only available in the Enterprise edition of Corda." href="https://docs.r3.com/en/platform/corda/5.0/introduction/about-the-docs.html#documenting-product-editions" class="enterprise-icon">
   <img src="/icons/enterprise-icon.png" alt="enterprise-icon"  width="28px" />
```
Comment (yarivus) [inline themes/doks/layouts/shortcodes/enterprise-icon.html:13]: if you accept my previous comment - needs changing here

File: content/en/platform/corda/5.0/introduction/about-the-docs.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## Documenting Product Editions
+The documentation relating to Corda 5 is structured differently than in previous versions of Corda. Previously, Corda Enterprise and Corda Community content was published in separate documentation sets. 
+From Corda 5.0 onwards, there is one documentation set, but the features that relate to Corda Enterprise are marked to avoid confusion. 
+
+{{< 
+  figure
+	 src="enterprise-icon.png"
+   width=55%
+	 figcaption="Documentation heading with Enterprise icon"
```
Comment (yarivus) [inline content/en/platform/corda/5.0/introduction/about-the-docs.md:25]: see my next comment about the filename `enterprise-icon.png`

File: content/en/platform/corda/5.0/introduction/about-the-docs.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+	 alt="Enterprise icon"
+>}}
+
+* **(A)** The heading without the ‘Enterprise only' icon is the most common and it is used where the content relates to both Corda Community and Corda Enterprise editions of the product.
+* **(B)** The ‘Enterprise only' icon is used mostly in headings, indicating that the entire content under that heading relates only to Corda Enterprise. On occasions, the icon may be used inline to identify sentences, paragraphs, admonitions, or tables, that are for Corda Enterprise only.
```
Comment (yarivus) [inline content/en/platform/corda/5.0/introduction/about-the-docs.md:30]: I'd find a more common synonym to `admonitions`

**Review by yarivus — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.0/introduction/enterprise-icon.png
Comment (colin-hillis) [reply content/en/platform/corda/5.0/introduction/enterprise-icon.png:1]: fair point - changed to:

**docs-enterprise-icon-use-case.png**

File: content/en/platform/corda/5.0/introduction/about-the-docs.md (hunk 3 above)
Comment (colin-hillis) [reply content/en/platform/corda/5.0/introduction/about-the-docs.md:25]: Changed this one.

File: themes/doks/layouts/shortcodes/enterprise-icon.html (hunk 2 above)
Comment (colin-hillis) [reply themes/doks/layouts/shortcodes/enterprise-icon.html:13]: This is not the same screenshot (.png) that I have renamed. This relates to the previous merge by Zheng. This is the class used for the actual icon that he created, so this one is ok.

File: content/en/platform/corda/5.0/introduction/about-the-docs.md (hunk 4 above)
Comment (colin-hillis) [reply content/en/platform/corda/5.0/introduction/about-the-docs.md:30]: Just removing it. Paragraphs loosely covers it, as there is no other common single word to replace it.

File: content/en/platform/corda/5.0/introduction/about-the-docs.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+In 2023, the R3 Technical Writing team will be making big changes to our documentation and our overall content strategy. As we make improvements, we’d love to get your feedback on what we’re doing well, and what we could be doing better. 
+
+Click [HERE](mailto:docs@r3.com) to email the R3 Technical Writing team. To help us deal with your feedback effectively, please include all of the following:
+
+* Your name
+* Your company name
+* Your email address (if different from the one being used to report)
+* The URL of the documentation page (if it is an issue being reported)
+
+Please include as much detail as you can in your feedback, as it is valuable to us and it will be read and shared with the rest of the team. JIRA tickets are created for any issues received, which will be quickly triaged and if necessary, assigned to a Technical Writer to effect changes.
```
[the remark this answers is not in the document]
Comment (colin-hillis) [reply content/en/platform/corda/5.0/introduction/about-the-docs.md:59]: Done - only for the purposes of not disclosing our tooling.

**Review by yarivus — DISMISSED** [review]
LGTM

**Review by yarivus — DISMISSED** [review]
LGTM

**Review by nadinequinr3 — APPROVED** [review]

**Pull request merged**